File: examples/max6/other/d_FirstBlink.py
'''
When launched, this script turns on all LEDs.
'''

from net.loadbang.shado import Block
from net.loadbang.shado.types import LampState
from config import renderer

lightBlock = Block(8, 8)
lightBlock.fill(LampState.ON)
renderer.render(lightBlock)

File: examples/max6/other/d_FirstBlink.clj
(ns user
  (:import (net.loadbang.shado Block))
  (:import (net.loadbang.shado.types LampState))
  (:require [config :as c]))

(let [lightBlock (Block. 8 8)]
  (.fill lightBlock LampState/ON)
  (.render c/renderer lightBlock))
